pandas/core/window/rolling.py

```diff
@@ -897,6 +897,32 @@ class Window(BaseWindow):
     2013-01-01 09:00:03  3.0
     2013-01-01 09:00:05  NaN
     2013-01-01 09:00:06  4.0
+
+    Rolling window apply function that uses multiple columns as input using
```

Comment:
This might be better in the rolling apply docstring.

Reply:
The doc example is primarily about how to use the `method='table'` feature in the rolling function. Also the `apply` doc currently doesn't include any examples - all the examples seem to be in the rolling doc (e.g. `.sum(std=3)`)

Reply:
Right but the particular usage of `method='table'` feature in the rolling function is only specific to `rolling(...).apply(...)`.

The `apply` doc could definitely use more examples!

Reply:
Would you like me to move this to the apply docstring then?

Reply:
Yes, this example you made would go great in the rolling apply docstring

Reply:
@mroeschke Hi, I've tried adding the docs to the apply doc-string but I'm unsure how to do it - my latest commit isn't working. Sorry I am unfamiliar with how the doc-string is being generated.

Reply:
You can see in this CI check what modifications need to be made to get this check to pass: https://github.com/pandas-dev/pandas/pull/40519/checks?check_run_id=2342425934

Additionally you can see here how to render the docs locally to see if there's inconsistent formatting somewhere: https://pandas.pydata.org/pandas-docs/dev/development/contributing_documentation.html#building-the-documentation